node/src/services/blockchain/commands.rs

```diff
@@ -0,0 +1,121 @@
+use anyhow::Result;
+use serde_json::Number;
+use sp_core::H256;
+use storage_hub_infra::actor::ActorHandle;
+
+use super::{
+    handler::BlockchainService,
+    types::{Extrinsic, ExtrinsicResult},
+};
+
+/// Commands that can be sent to the BlockchainService actor.
+#[derive(Debug)]
+pub enum BlockchainServiceCommand {
+    SendExtrinsic {
+        call: storage_hub_runtime::RuntimeCall,
+        callback: tokio::sync::oneshot::Sender<Result<(tokio::sync::mpsc::Receiver<String>, H256)>>,
+    },
+    GetExtrinsicFromBlock {
+        block_hash: H256,
+        extrinsic_hash: H256,
+        callback: tokio::sync::oneshot::Sender<Result<Extrinsic>>,
+    },
+    UnwatchExtrinsic {
+        subscription_id: Number,
+        callback: tokio::sync::oneshot::Sender<Result<()>>,
+    },
+}
+
+/// Interface for interacting with the BlockchainService actor.
+pub trait BlockchainServiceInterface {
+    /// Send an extrinsic to the runtime.
+    async fn send_extrinsic(
+        &self,
+        call: impl Into<storage_hub_runtime::RuntimeCall>,
+    ) -> Result<(tokio::sync::mpsc::Receiver<String>, H256)>;
```

Comment:
The return here is a string and a hash, but what do they represent? Maybe a type alias to make it clearer could work, or adding it to the docs of the function